crates/mirror-mirror/src/foreign_impls/mod.rs

```diff
@@ -80,3 +80,52 @@ __private_derive_reflect_foreign! {
         end: Idx,
     }
 }
+
+#[cfg(feature = "glam")]
+mod glam_impls {
```

Comment:
Would it make sense to put this in an `glam.rs` file? It's a bit weird to have it in the `mod.rs` module here (I did look for it in another file when trying that PR locally.)